src/tests/JIT/HardwareIntrinsics/X86_Avx512/Avx512F/EmbeddedRounding.Double.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We likely want to make this a new test template if possible for the same reasons as above. We have a large number of instructions this applies to and we really want to be adding basic tests for most of them.

Actually testing that two arbitrary floats/doubles are added and round correctly is likely non-trivial, however. I don't think this test is validating that so much as its just showing it doesn't explode for such inputs.

Given its testing `ToEven`, it's not really testing the new code paths either. That is, its not validating anything around the embedded rounding actually being used.

Reply:
Yes, this test was initially to put a draft here which just tests the compiling path instead of the results. I was having some difficulty to manually calculate the expected results under difference rounding mode settings.